.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,120 @@
+name: Node.js CI
+
+on: [push, pull_request]
+
+jobs:
+  test-build-release:
+
+    runs-on: ubuntu-latest
+    # Add postgres for end-to-end
+    services:
+      postgres:
+        image: postgres:10.8
+        # Set postgres env variables according to test env.yml config
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: catalogue
+        ports:
+          - 5432:5432
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
```

Comment:
I just came across [this AWS announcement regarding Python versions](https://docs.aws.amazon.com/cli/latest/userguide/welcome-versions.html). It is not a blocking item, however if we proceed as written, we should also open an issue to install Python 3.6+.

Reply:
Disregards, the v2 tag on the action is misleading...